tests/rpc_performance_test.ts

```diff
@@ -16,23 +16,30 @@ const test_config = {
         vn_batch_number: 2,
         vn_batch_size: 4096,
     },
-    timeout: 200000,
-    expected_sync_time: 600
+    timeout: 1000000,
```

Comment:
looks like you can calculate the timeout from `expected_sync_time` * `expected_average_sync_time` * (`deviation_percentage`+1) instead of hardcoding it

Reply:
I can't have it in this const struct if I want to make it a variable. The current timeout should give enough time for everything to finish so the prints can be logged.

Reply:
exactly my point. if it can be calculated, it shouldnt be an excplicitly hardcoded constant

>  The current timeout should give enough time for everything to finish so the prints can be logged.

Until you decided to run 50 more/less sync loops for some reason and forget to adjust the timeout

Reply:
sure

Reply:
Actually you can't calculate it like this, because as you can see from the code there's a storage setup phase.

Reply:
From what I can see this is included in `expected_average_sync_time`, no?

Reply:
The expected_average_sync_time is total_sync_time / total_iterations, different than the timeout for the whole test.

Reply:
different how and/or why?

Reply:
because the test has to setup the storage first

Reply:
Let me include this in the description. I kind of skipped over these details because the tests were already merged from a dfifferent PR.

Reply:
Not knowing the test framework, i assume the setup happens in that line

https://github.com/Manta-Network/Manta/pull/681/files/c8261bf4b29f8a1ef09386b96221c15a481d6ce5..43b1de6efd1c84c391b306359db3aa197c3fb0ed#diff-19d51960f59786f9c40532e38e3a902c64be0e21e3b84a46702bcb0718a20605R29

If that's not the case and setup is a one-time invocation that's constant, can't you just add that constant to the calculated timeout?

Reply:
It's not constant.

Anyway, there can be two timeouts, one that is eye-balling the storage injection plus expected total sync time.